tests/acceptance/steps/min_isr.py

```diff
@@ -37,7 +39,10 @@ def set_min_isr(topic, min_isr):
     cluster_config = get_cluster_config()
     with ZK(cluster_config) as zk:
         config = zk.get_topic_config(topic)
-        config['config'] = {ISR_CONF_NAME: str(min_isr)}
+
+        serialized_min_isr = str(min_isr)
```

Comment:
I think I made this change for another thing that is no longer relevent. Undone!